cl/opinion_page/templates/docket_tabs.html

```diff
@@ -71,33 +79,33 @@ <h1 class="bottom inline">
 
     <h2>{{ docket.court }}</h2>
 
-    {% if docket.pacer_url or docket_entries %}
+    {% if docket.pacer_docket_url or docket_entries %}
       <div class="v-offset-above-2">
-        {% if docket.pacer_url %}
-          <div class="left">
-            <p><a href="{{ docket.pacer_url }}"
-               class="btn btn-primary"
-               target="_blank"
-               rel="nofollow"><i class="fa fa-external-link"></i> View on PACER</a>&nbsp;</p>
-          </div>
-        {% endif %}
         {% if docket_entries %}
-          <div class="left">
-            <p><a href="/?type=r&amp;q=docket_id%3A{{ docket.pk }}"
-               class="btn btn-success"><i class="fa fa-search"></i>&nbsp;Search this Docket</a>&nbsp;</p>
-          </div>
+          <a href="/?type=r&amp;q=docket_id%3A{{ docket.pk }}"
+             class="btn btn-success"><i class="fa fa-search"></i>&nbsp;Search this Docket</a>
         {% endif %}
-        <div class="left">
+        <div class="btn-group"
+             id="react-root"
+             data-docket="{{ docket.id }}"
+             data-user-id="{% if user.is_authenticated %}{{ user.pk }}{% endif %}"
+             data-user-name="{% if user.is_authenticated %}{{ user.get_username }}{% endif %}"
+             data-authenticated="{% if user.is_authenticated %}{{ user.pk }}:{{ user.get_username }}{% endif %}"
```

Comment:
Sorry, if I missed it, but I can't find where this is being used anywhere. Is it possible we don't use it? If we don't, let's remove it from here and elsewhere. Are all the others in use wherever we have react-root?

Reply:
In this instance -- 

The data attributes get automatically transferred and converted to camel case format.

`const TagSelect: React.FC<UserState> = ({ userId, userName, editUrl, docket }) => {
`